### ISSUE TITLE
Issues building convpot in linux

### ISSUE THREAD

**Issue by darkwraith** [ahpohl/convpot#5]
I am trying to build a convpot for linux and I'm not sure if there is something I'm doing wrong but I wanted to share my issue. 
My cmake .. -G "Unix Makefiles" completes fine but when I perform the "make" I get this error "fatal error: sqlext.h: No such file or directory"

Any help would be appreciated. 

Scanning dependencies of target sqlite3
[  5%] Building C object sqlite/CMakeFiles/sqlite3.dir/src/sqlite3.c.o
[ 11%] Linking C static library libsqlite3.a
[ 11%] Built target sqlite3
Scanning dependencies of target convpot
[ 16%] Building CXX object CMakeFiles/convpot.dir/src/arbin.cpp.o
In file included from /home/jim/Downloads/convpot-master/src/arbin.cpp:2:0:
/home/jim/Downloads/convpot-master/include/device.h:12:20: fatal error: sqlext.h: No such file or directory
compilation terminated.
CMakeFiles/convpot.dir/build.make:62: recipe for target 'CMakeFiles/convpot.dir/src/arbin.cpp.o' failed
make[2]: *** [CMakeFiles/convpot.dir/src/arbin.cpp.o] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/convpot.dir/all' failed
make[1]: *** [CMakeFiles/convpot.dir/all] Error 2
Makefile:151: recipe for target 'all' failed
make: *** [all] Error 2

**Comment by darkwraith** [comment]
Made a little progress after posting this. Seems part of the issue is that you need unixodbc and unixodbc-dev packages to provide proper linking. I'm still getting some issues with an improperly simlinked library but will have to attempt again later.

**Comment by ahpohl** [comment]
Linux build issues are fixed now. I've created a package for Arch linux and uploaded Convpot to the AUR. Instruction how to build and install the package are added to the [wiki](https://github.com/ahpohl/convpot/wiki/Building-and-installing-under-Arch-Linux). If you are using Arch linux, I would be very grateful if you have the time to test the new package and post your experience to this thread.

The CMake build should work also on any Linux distribution. Package dependencies are sqlite3, unixodbc and mdbtools. Will try my best to also create a Debian/Ubuntu package with CPack.

**Comment by ahpohl** [comment]
close this issue as CMake makefile has been created for linux support.

**Comment by darkwraith** [comment]
Unfortunately I dont have an Arch system. Currently running linux mint 18.1 64 bit. I think I might have gotten it to build and package for debian.

Some issues I ran into: the cmake required installing sqlite3, and/or libsqlite3-dev to avoid getting a "sqlite3.h not found error". Also the cmake was not able to automatically find the correct "CMakeLists.txt.linux" file. I had to manually move/rename it to the root folder. 

Also in able to do the CPack, I had to add this to the bottom of the CMakeLists.txt.linux file:
SET(CPACK_GENERATOR "DEB")
SET(CPACK_DEBIAN_PACKAGE_MAINTAINER "ahpohl") #required
 
INCLUDE(CPack)
